scripts/bootstrap-aio.yml

```diff
@@ -142,21 +142,27 @@
 
     - name: Add target user vars to user_variables_overrides
       set_fact:
-        user_variables_overrides: "{{ user_variables_overrides|combine(target_user_variables_overrides[TARGET])}}"
+        user_variables_overrides: "{{ user_variables_overrides|combine(target_user_variables_overrides[target])}}"
       when:
-        - TARGET is defined
-        - TARGET in target_user_variables_overrides
+        - target is defined
+        - target in target_user_variables_overrides
 
   vars:
     rpco_deploy_hardening: "{{ lookup('env', 'DEPLOY_HARDENING') }}"
     TRIGGER: "{{ lookup('env', 'TRIGGER')}}"
     SCENARIO: "{{ lookup('env', 'SCENARIO')}}"
-    TARGET: "{{ lookup('env', 'TARGET')}}"
+    target: "{{ TARGET | default(lookup('env', 'TARGET')) }}"
 
 
 - name: Execute the OSA AIO bootstrap
-  include: "{{ lookup('env', 'OA_DIR') }}/tests/bootstrap-aio.yml"
+  hosts: localhost
+  user: root
+  tasks:
+   - include: "{{ lookup('env', 'OA_DIR') }}/tests/bootstrap-aio.yml"
```

Comment:
Ah, so I actually can use `when` on playbook includes because that was added in ansible 2.0. It'll be trickier for the branches with older versions of ansible, but I can address that in the future, I suppose.